### ISSUE TITLE
Announce dynamic content updates to screen reader users

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#4064]
### Describe the Problem

Currently dynamic content updates are not announced to screen readers, for ex. contrast mode being changed, link being copied, language or city being changed and so on.

### Describe Your Preferred Solution

Check where it is necessary and apply screen-reader announcement

### Describe Possible Alternatives

N/A

### Related Issues

similar to #3984
